unit-tests/src/org/commcare/android/tests/queries/CaseDbQueryTest.java

```diff
@@ -66,7 +66,19 @@ public void testCaseIndexQueries() {
                 "test_case_id_child_2", ec);
     }
 
-    public static void evaluate(String xpath, String expectedValue, EvaluationContext ec) {
+    @Test
+    public void testCaseOptimizationTriggers() {
```

Comment:
I don't actually think we can, since I'm not sure what the SQL-based optimization status is on Android. CloudCare wasn't demonstrating the bug, for instance.